spec/models/user_spec.rb

```diff
@@ -1,5 +1,74 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user) { build(:user) }
+  let!(:next_user) { create(:user) }
+  let(:attributes_presence) { %i[first_name last_name phone project_id email] }
+
+  context 'when got project id' do
+    it 'should get error message if id absent' do
+      id = 22_423
+      expect(Project.exists?(id)).to eql(false)
```

Comment:
Where is your expected error message?
Your title doesn't match your expectation.

Reply:
You shouldn't use random id to check user's id. It totally wrong test